.eslintignore

```diff
@@ -0,0 +1,3 @@
+.nyc_output/
+coverage/
+node_modules/
```

Comment:
>>> In addition to any patterns in a .eslintignore file, ESLint always ignores files in /node_modules/* and /bower_components/*.

In addition to any patterns in a .eslintignore file, ESLint always ignores files in /node_modules/* and /bower_components/*.

Reply:
Thanks! Removed.